### ISSUE TITLE
Long RRule Schedule Display

### ISSUE THREAD

**Issue by vdytyniak-exos** [PrefectHQ/prefect#7663]
### First check

- [X] I added a descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I refreshed the page and this issue still occurred.
- [X] I checked if this issue was specific to the browser I was using by testing with a different browser.

### Bug summary

Long RRule schedule adds horizontal scroll, so it is hard to reach and see other Deployment fields. 

### Reproduction

Deploy long RRule Schedule. 

### Error

<img width="1186" alt="image" src="https://user-images.githubusercontent.com/94358054/204225913-e648382e-a6cb-4ceb-8031-0eae9b9e5efe.png">

### Browers

- [X] Chrome
- [ ] Firefox
- [X] Safari
- [ ] Edge

### Prefect version

```Text
Version:             2.6.9
API version:         0.8.3
Python version:      3.10.5
Git commit:          014d093e
Built:               Tue, Nov 22, 2022 2:14 PM
OS/Arch:             darwin/x86_64
Server type:         cloud
```

**Comment by anna-geller** [comment]
could you share a bit more about your use case? how did you create that schedule? 

if you want to create a bunch of scheduled runs for specific dates, you could loop over those dates and pass them to the `run_deployment` utility

**Comment by vdytyniak-exos** [comment]
It is simple `RRuleSchedule` with list of dates using `RDATE:` keyword. I don't want to manage schedule using run_deployment. This issue is just about UX.

**Comment by vdytyniak-exos** [comment]
We need to run the flow using custom calendar, that is why we pass list of RDATEs.

**Comment by jakekaplan** [comment]
@vdytyniak-exos this doesn't exactly solve the display issue but theres a pr open to limit the max size of rrules on deployments in general https://github.com/PrefectHQ/prefect/pull/7762. In case your rrules are particularly large, it may make sense to convert to a smaller/different schedule, maybe using RRULE/EXRULE or EXDATES

**Comment by vdytyniak-exos** [comment]
It is not possible to describe with RRule. In Prefect 1 we used DatesClock that is not supported anymore. Would be great to provide optional description for schedule and display it instead of long rrule.

**Comment by anna-geller** [comment]
Could you say a bit more about that custom calendar? How do you define that in rrule? It would be great to collect more info before we can give any concrete recommendations and decide about the next steps

**Comment by bphillips-exos** [comment]
Hi Anna, we work in the finance industry and our flow schedules are often tied to the holiday calendars for different countries or markets. The example I gave on Jake's PR was that we have one flow that needs to run according to the French bond market calendar and another according to Malaysian Bank Holidays. We have a vendor that provides data on these different calendars, but the data is in the form of a list of business days, not RRULEs. Currently we are using a list of RDATEs to provide scheduling information to Prefect. As I mentioned on Jake's PR, we are evaluating whether we can switch to exclusionary EXDATEs and an RRULE for weekends to shorten the overall RRULE string length. In either case, the resulting RRULE string is not at all human readable but we could provide a very succinct description like "French Bond Trading" that would be useful for users in the UI. 

We welcome other suggestions or ideas, I personally had not used RRULEs until they became the only way we know of to schedule flows with custom calendars on Prefect so it's entirely possible we are missing a more obvious solution 🙂

**Comment by anna-geller** [comment]
yeah exactly, I'm also curious about what's the right to do it. There must be some way to provide just the time zone of that specific country and possibly a list of dates to exclude rather than manually specifying every scheduled date. It seems quite tedious and error-prone. Let me ask the team if they are aware of a better way to approach it. Thanks so much for sharing your use case

**Comment by zhen0** [comment]
Thanks for digging into the bigger issues here Anna.  Regardless of how we proceed, the ui should handle long strings like this better and will need to truncate whatever is shown in the deployments table more than in https://github.com/PrefectHQ/prefect/pull/7762.

**Comment by zhen0** [comment]
https://github.com/PrefectHQ/orion-design/pull/868 improves the UX for handling long rrule schedules - it removes the long string in the deployment table and shows 'RRule' it its place.  The details of the schedule are still shown on hover and in the deployment details page. 

In slower time, we will investigate a better ui library for rrule handling and/or see if previous issues in rrule.js have been resolved.

<img width="917" alt="image" src="https://user-images.githubusercontent.com/40272060/205752935-103367b4-2378-4466-9dd7-90cb249f880a.png">

**Comment by zhen0** [comment]
Marking as closed as the fix for the ux issue of a long string is now released in both open source and cloud.

**Comment by discdiver** [comment]
With Orion on 2.7.6 I'm not seeing RRule expansion on hover. Would you like me to open a new issue @zhen0?
